src/diff-snapshot.js

```diff
@@ -184,6 +184,24 @@ const shouldFail = ({
   };
 };
 
+function composeDiff(options) {
+  const {
+    diffDirection, baselineImage, diffImage, receivedImage, imageWidth, imageHeight, onlyDiff,
+  } = options;
+  const composer = new ImageComposer({
+    direction: diffDirection,
+  });
+
+  if (!onlyDiff) {
+    composer.addImage(baselineImage, imageWidth, imageHeight);
+  }
+  composer.addImage(diffImage, imageWidth, imageHeight);
+  if (!onlyDiff) {
+    composer.addImage(receivedImage, imageWidth, imageHeight);
+  }
+  return composer;
```

Comment:
I created a new function, otherwise the complexity of `diffImageToSnapshot` was too high